docs/workflows/doi.md

```diff
@@ -1,4 +1,120 @@
-# DOI
+# DOI Workflow
 
-Add Workflow Description here
+The primary purpose of the DOI workflow is to link together open datasets of higher education through the use of persistent identifiers, and to then produce useful summary statistics across a variety of common aggregations
 
+## Input datasets
+
+Each a number of specialised research publication datasets are collected. These include Crossref Metadata, Crossref Funder Registry, Crossref Events, Microsoft Academic Graph, Unpaywall, the Research Organization Registry, Open Citations and Geonames. The table below illustrates each of these datesets and the PIDs that are exposed within them.
+
+```eval_rst
++-------------------+-----------------------------------------------------+-----------+
+| Name              | Table                                               | PID       |
++===================+=====================================================+===========+
+| Crossref Metadata | `crossref.crossref_metadataYYYYMMDD`                | DOI       |
++-------------------+-----------------------------------------------------+-----------+
+| Crossref Events   | `crossref.crossref_events`                          | DOI       |
++-------------------+-----------------------------------------------------+-----------+
+| Crosref Fundref   | `crossref.crossref_fundrefYYYYMMDD`                 | FunderDOI |
++-------------------+-----------------------------------------------------+-----------+
+| Grid              | `digital_science.gridYYYYMMDD`                      | GRID      |
++-------------------+-----------------------------------------------------+-----------+
+| MAG               | `mag.*YYYYMMDD` (multiple individual tables)        | GRID/DOI  |
++-------------------+-----------------------------------------------------+-----------+
+| Open Citations    | `open_citations.open_citationsYYYYMMDD`             | DOI       |
++-------------------+-----------------------------------------------------+-----------+
+| ORCID             | `orcid.orcid`                                       | ORCID/DOI |
++-------------------+-----------------------------------------------------+-----------+
+| Unpaywall         | `our_research.unpaywall`                            | DOI       |
++-------------------+-----------------------------------------------------+-----------+
+| iso3166           | `ios.iso3166_countries_and_regions`                 | Country   |
++-------------------+-----------------------------------------------------+-----------+
+```
```

Comment:
Some minor comments:
- Is the Geonames table the same as the iso3166 table? Or how is this table derived from Geonames?
- Grid is listed, but ROR isn't, I understand we're in the process of transitioning between them, but it is a bit confusing now as ROR is listed in the diagram below, while GRID isn't.
- ROR and/or GRID should also have 'Country' as a PID I think? If I understand correctly that this is how an item is linked to a specific country from iso3166 dataset.

Reply:
Thanks for the feedback! I have added a section on the dashboard and data_export datasets. Now that RoR is merged, I have updated it all to only reference RoR. I also added country as a PID as I think you are right, and that is consistent with how I was using it for other datasets